cmd/create/service/cmd.go

```diff
@@ -315,78 +316,38 @@ func run(cmd *cobra.Command, argv []string) {
 
 	role := aws.AccountRoles[aws.InstallerAccountRole]
 
-	roleARNs, err := r.AWSClient.FindRoleARNs(aws.InstallerAccountRole, minor)
+	// Find all installer roles in the current account using AWS resource tags
+	currentArnBundles, err := r.AWSClient.FindAccountRoleArnBundles(minor)
 	if err != nil {
 		r.Reporter.Errorf("Failed to find %s role: %s", role.Name, err)
 		os.Exit(1)
 	}
 
-	if len(roleARNs) > 1 {
-		defaultRoleARN := roleARNs[0]
-		// Prioritize roles with the default prefix
-		for _, rARN := range roleARNs {
-			if strings.Contains(rARN, fmt.Sprintf("%s-%s-Role", aws.DefaultPrefix, role.Name)) {
-				defaultRoleARN = rARN
-			}
-		}
-		r.Reporter.Warnf("More than one %s role found, using %q", role.Name, defaultRoleARN)
-		roleARN = defaultRoleARN
-	} else if len(roleARNs) == 1 {
-		if !output.HasFlag() || r.Reporter.IsTerminal() {
-			r.Reporter.Infof("Using %q for the %s role", roleARNs[0], role.Name)
-		}
-		roleARN = roleARNs[0]
-	} else {
-		r.Reporter.Errorf("No account roles found. " +
+	if len(currentArnBundles) == 0 {
+		r.Reporter.Warnf("No account roles found. " +
```

Comment:
Should we specify bundles?
Because the user might have plenty of account roles.
```suggestion
		r.Reporter.Warnf("No account roles bundles found. " +
```

Reply:
Makes sense, will do